tms/clickup/constant.py

```diff
@@ -0,0 +1 @@
+password_not_match = "Password not Match."
```

Comment:
PASSWORD_ERROR_MESSAGE = "............."
from next time use variable name like this.